floss/main.py

```diff
@@ -370,10 +380,11 @@ def load_vw(
     sigpaths: List[Path],
     should_save_workspace: bool = False,
 ) -> VivWorkspace:
+    file_type = get_file_type(sample_path)
     if format not in ("sc32", "sc64"):
-        if not is_supported_file_type(sample_path):
+        if file_type is None:
```

Comment:
then adjust the logic here accordingly, could check `UNSUPPORTED_FILE_MAGIC` first

Reply:
@mr-tz , For format = "auto" this will raise an error should i add "auto" in conditional check as well?

Reply:
ah right, then we can leave the order